Laundromat/pom.xml

```diff
@@ -8,7 +8,7 @@
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
         <version>3.0.6</version>
-        <relativePath/> <!-- lookup parent from repository -->
+        <relativePath/>
```

Comment:
Возникает ошибка: `points at org.ds:Laundry instead of org.springframework.boot:spring-boot-starter-parent, please verify your project structure`

Этот тег нужен, т.к. мы подключаем в качестве родительского проекта spring-boot и у нас Laundromat дочерний модуль Laundry, то есть зависимости подтягиваются из двух модулей.